client/src/templates/Introduction/Intro.js

```diff
@@ -83,7 +83,7 @@ IntroductionPage.propTypes = propTypes;
 export default IntroductionPage;
 
 export const query = graphql`
-  query IntroPageBySlug($slug: String!, $block: String!) {
+  query IntroPageBySlug($block: String!) {
     markdownRemark(fields: { slug: { eq: $slug } }) {
       frontmatter {
         block
```

Comment:
```suggestion
    markdownRemark(fields: { slug: { eq: $slug } }) {
      frontmatter {
        block
```
As I mentioned above, these shouldn't be necessary once the markdown files are gone.